commands/queue.js

```diff
@@ -0,0 +1,44 @@
+const {GuildMember} = require('discord.js');
+const { slice } = require('ffmpeg-static');
```

Comment:
Why are you importing `slice` from `ffmpeg-static` here? Is it an accidental import statement since you are using `str.splice` below?

Reply:
That is my bad I don't know how that ended up in there I didn't write that statement but I have removed and confirmed it works as intended on my instance.